### DISCUSSION TITLE
Unable to get only the value of a sensor through the API

### DISCUSSION THREAD

**Discussion by Kuvesz** [nicolargo/glances#2623] [General]
#### Description

Hi! This may be a stupid question but I'm not really sure how I'm supposed to get only the `value` field from a response like the one I posted below. Any help regarding this would be appreciated!

#### Versions

* Glances & psutil versions: `Glances v3.2.4.2 with PsUtil v5.8.0`
* Operating System: `Debian 11`
* How do you install Glances (Pypi package, script, package manager, source): `Using the docker image`

#### Configuration and log file

```
curl http://localhost:61208/api/3/sensors/label/Composite

{"Composite": [{"label": "Composite", "value": 36, "warning": 74, "critical": 79, "unit": "C", "type": "temperature_core", "key": "label"}]}
```

**Comment by nicolargo** [comment]
Hi @Kuvesz 

The Glances API is designed to return a key: value.

Nevertheless, you can use a simple jq command to extract the value.

Example with key: value:

```
curl --no-progress-meter http://0.0.0.0:61208/api/3/sensors/label/CPU | jq .
{
  "CPU": [
    {
      "label": "CPU",
      "unit": "C",
      "value": 55,
      "warning": null,
      "critical": null,
      "type": "temperature_core",
      "key": "label"
    }
  ]
}
```

Example with values (list):

```
curl --no-progress-meter http://0.0.0.0:61208/api/3/sensors/label/CPU | jq .CPU
[
  {
    "label": "CPU",
    "unit": "C",
    "value": 57,
    "warning": null,
    "critical": null,
    "type": "temperature_core",
    "key": "label"
  }
]
```

Example with value (first item of the list):

```
curl --no-progress-meter http://0.0.0.0:61208/api/3/sensors/label/CPU | jq .CPU[0]
{
  "label": "CPU",
  "unit": "C",
  "value": 55,
  "warning": null,
  "critical": null,
  "type": "temperature_core",
  "key": "label"
}
```
